apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -2097,6 +2103,270 @@
       "MERGE_OUTPUTS": "Minimize future fees (merge coins)"
     }
   },
```

Comment:
Ledger team - is there anything else I need to do for the other localizations?

Reply:
Hey @andyhass nope that's perfect, only English is required, translations are automatically provided afterwards by a dedicatd team. :slightly_smiling_face:

Reply:
Hey @andyhass nope that's perfect, only English is required, translations are automatically provided afterwards by a dedicated team. slightly_smiling_face